src/pybanana/__init__.py

```
@@ -13,43 +13,56 @@
+    PcSpecs,
+    SoftwareKit,
+    GamingDevices,
     File,
     LicenseChecklist,
     Medals,
     OnlineStatus,
-    ManagerRecord, 
-    ModeratorRecord,
-    DiscussionRecord,
-    
+    Manager,
```

Ruff (0.12.2)

Redefinition of unused `Manager` from line 18

Remove definition: `Manager`

(F811)